src/layout/navbar/NavBar.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect } from 'react';
+import { Tooltip, UnstyledButton, Stack, rem } from '@mantine/core';
+import { IconHome2, IconGauge, IconFingerprint, IconLogout } from '@tabler/icons-react';
+import classes from './NavBar.module.css';
+import { RoutePaths } from 'Enum/Enum';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useAuth } from 'hooks/useAuth';
+import { persistor, useAppDispatch } from 'store/store';
+import { logout } from 'redux/slice/userSlice';
+interface NavbarLinkProps {
```

Comment:
add emty line and use type Props for all component:
```suggestion
type Props = {
```